src/frontend/src/components/public/PIAFormTabs/review/index.tsx

```diff
@@ -69,6 +78,44 @@ const PIAReview = ({ printPreview }: IReviewProps) => {
     setReviewForm(pia.review);
   }, [pia.review]);
 
+  /**
+   * Update reviewForm.programArea when
+   * Part 4 Assessment(storing personal information tab) (PIDSOC), If Assessment of Disclosures Outside of Canada is filled out in PIA,
+   * ADM(Assistant Deputy Minister) is a preselected role
+   * and can not be deleted
+   * personalInformation.storedOutsideCanada is yes
+   * sensitivePersonalInformation.doesInvolve is yes
+   * sensitivePersonalInformation.disclosedOutsideCanada is no
+   */
+  useEffect(() => {
+    // if the condition does satisfy the rule, add the adm to programArea
+    // otherwise do nothing
+    if (
+      pia?.storingPersonalInformation?.personalInformation
+        ?.storedOutsideCanada === YesNoInput.YES &&
+      pia?.storingPersonalInformation?.sensitivePersonalInformation
+        .doesInvolve === YesNoInput.YES &&
+      pia?.storingPersonalInformation?.sensitivePersonalInformation
+        .disclosedOutsideCanada === YesNoInput.NO
```

Comment:
Are we sure this has to NO?

Reply:
yes, 
<img width="1136" alt="Screenshot 2023-07-18 at 12 05 13 PM" src="https://github.com/bcgov/cirmo-dpia/assets/57013495/690ad2bc-ddab-41c5-acfd-03410e73046d">